app/tests/test_admin_page.py

```diff
@@ -53,6 +53,7 @@ def _create_collection(self, with_link=False, with_provider=False, extra=None):
             "name": "test_collection",
             "license": "free",
             "description": "some very important collection",
+            "published": "on",
```

Comment:
Is it really `on` here, or should that be `true`?

Reply:
Yes and to disable it is not `"off"` like everyone would thought, but the field needs to be removed from the payload ;-)